step.yml

```diff
@@ -2,31 +2,52 @@ title: "Deploy to Bitrise.io - Apps, Logs, Artifacts"
 summary: |-
   Deploys build artifacts to make them available for the user on the build's **APPS & ARTIFACTS** tab. Sends test results to the Test Reports add-on.
 description: |-
-  The Step collects artifacts from previous Steps in the Workflow and places them in a specified deploy directory or compresses them into a single .zip file in the deploy directory. These artifacts are then available on the **APPS & ARTIFACTS** tab of any given build. For installable artifacts, such as IPAs or APKs, the Step can create a public install page that allows testers to install the app on their devices. You can also use the Step to notify users about the build.
-
-  This Step is required in order to use the Test Reports add-on. It converts the test results to the right format and sends them to the add-on.
+  The Step accesses artifacts from a directory specified as the `$BITRISE_DEPLOY_DIR` where artifacts generated by previous Steps gets stored. 
+  These artifacts are then uploaded on the **APPS & ARTIFACTS** tab of any given build. For installable artifacts, such as IPAs or APKs, the Step can create a public install page that allows testers to install the app on their devices. 
+  You can also use the Step to notify users about the build. If you wish to use the Test Reports add-on, you must add this Step in your Workflow since the Step converts test results to the right format and sends them to the add-on.
 
   ### Configuring the Step
 
-  1. Set the value for the Deploy directory of file path. The default is the $BITRISE_DEPLOY_DIR environment variable which is exposed by the Bitrise CLI.
-  2. Set the **Notify: User Roles** and the **Notify: Emails** inputs to configure notifications for the users. Note that the **Notify: Emails** input is only used if a public install page is created and you must use a Secret to set it.
-  3. Optionally, you can create a zip file for all your build artifacts by setting the **Compress the artifacts into one file?** input to `true`. 
-
-  Get a public install page that allows testers to install the app on their devices:
-
-  1. Set the **Enable public page for the App?** input to `true`.
-  2. Set the **Compress the artifacts into one file?** input to `false`. If all the contents of the deploy directory are compressed into a single zip file, there won't be a public install page.
+  1. Set the value for the **Deploy directory or file path** required input. It can be a path to a directory or a file. If you provide a directory, everything in that directory gets uploaded recursively. 
```

Comment:
`If you provide a directory, everything in that directory gets uploaded recursively.`

Sorry I gave you misinformation. I've double checked and if the directory is provided only files in the directory **excluding** the subdirectories will be uploaded.

Everything will be uploaded only if compression is enabled.

This is written well in the description of the input.

Reply:
Thanks @mateherber ! :). I'll correct it ;)